src/main/kotlin/day3/Slime.kt

```
@@ -0,0 +1,5 @@
+package day3
+
+class Slime(var hp: Int, val suffix: String) {
+    fun attack() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)